patient-service/src/main/java/com/raf/si/patientservice/mapper/HealthRecordMapper.java

```diff
@@ -134,4 +139,53 @@ private List<AllergyResponse> makeAllergyResponse(List<Allergy> allergies, Healt
 
         return allergyResponseList;
     }
+
+    public VaccineListResponse vaccineListToVaccineListResponse(List<Vaccine> vaccines) {
+        List<VaccineResponse> vaccineResponses = new ArrayList<>();
+        for(Vaccine vaccine : vaccines) {
+            VaccineResponse vaccineResponse = new VaccineResponse(
+                    vaccine.getId(),
+                    vaccine.getName(),
+                    vaccine.getType(),
+                    vaccine.getDescription(),
+                    vaccine.getProducer());
+            vaccineResponses.add(vaccineResponse);
+        }
+        return new VaccineListResponse(vaccineResponses);
+    }
+
+    public AllergenListResponse allergenListToVaccineListResponse(List<Allergen> allergens) {
```

Comment:
ime metode (ide u allergyListResponse). Mozes da ides i Intellij desni klik i da ides rename, to ce da promeni ime svuda gde se ime metode koristi (da ne moras rucno da menjas)